tests/integration/utils.py

```diff
@@ -180,6 +181,30 @@ def escape_path_go(dependency):
         return dependency
 
 
+def get_sha256_hash_from_file(filename):
+    """
+    Return sha256 hash of file.
+
+    :param str filename: The path to file
+    :return: sha256 hash of file
+    :rtype: str
+    """
+    # make a hash object
+    h = hashlib.sha256()
+
+    # open file for reading in binary mode
+    with open(filename, "rb") as file:
+        # loop till the end of the file
+        chunk = 0
+        while chunk != b"":
+            # read only 1024 bytes at a time
+            chunk = file.read(1024)
+            h.update(chunk)
```

Comment:
Not a big deal, but it feels weird for `chunk` to go from integer to bytes. Would this work instead?
```suggestion
        # loop till the end of the file, 1024 bytes at a time
        chunk = file.read(1024)
        while chunk:
            h.update(chunk)
            chunk = file.read(1024)
```
Or, with the "new", shiny python3.8 (but we can't use that yet :cry: )
```python
while (chunk := file.read(1024)):
    h.update(chunk)
```

Reply:
Thank you! I like your way more!